syntax = "proto3";

package RankProtocol;

enum ID {
    ZERO                    = 0;

    C2S_RANK_BASE           = 327680;   // ID_BASE + 1000   // 5 << 16
    C2S_RANK_LOAD           = 327681;
    S2C_RANK_LOAD           = 327682;
    C2S_RANK_UPDATE         = 327683;
    S2C_RANK_UPDATE         = 327684;
    C2S_RANK_TOP            = 328680;
}
